### DISCUSSION TITLE
재귀 관련한 학습 동영상 추천드립니다.

### DISCUSSION THREAD

**Discussion by brewagebear** [brewagebear/ISOLVEDTHEPROBLEM#29] [잡담]
https://www.youtube.com/watch?v=ln7AfppN7mY&list=PL52K_8WQO5oUuH06MLOrah4h05TZ4n38l&index=1 

시각화가 머리속에 잘안되서다시 공부를 해야겠구나 생각했는데 이 강의 좋은거 같네요.

**Comment by Tianea2160** [comment]
좋은 정보 감사드립니다!!
